spec/models/user_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
```

Comment:
- [ ] All of your tests are failing, please make sure they are passing. 

- In your database the column named as 'postsCounter', hovewer, in your model and tests you have used 'posts_counter'. That's why they are failing. 
![image](https://github.com/obibaadoma/Blog_App/assets/91486791/d19a7d21-c7cb-43eb-a35a-ecf8da94d460)